packages/QuietButton/QuietButton.jsx

```diff
@@ -82,7 +96,28 @@ QuietButton.propTypes = {
    * Button children.
    */
 
-  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(
+      PropTypes.oneOfType([
+        PropTypes.string,
+        componentWithName('A11yContent'),
+        componentWithName('Edit'),
+        componentWithName('Print'),
+        componentWithName('Profile'),
+        componentWithName('Add'),
+        componentWithName('Close'),
+        componentWithName('Delete'),
+        componentWithName('PlayVideo'),
+        componentWithName('Subtract'),
+        componentWithName('Download'),
+        componentWithName('DownloadPDF'),
+        componentWithName('DownloadPDFs'),
+        componentWithName('Search'),
+        componentWithName('Settings'),
```

Comment:
This can be simplified to:

```jsx
componentWithName('A11yContent'),
componentWithName('Dependent', true)
// setting the second parameter to `true` uses the component's `displayName`
```

The `displayName` for all Dependent icons are defined as 'Dependent', [see code](https://github.com/telus/tds-core/blob/8fb32ac8ebc037925e051f51070849651e5da542/packages/InteractiveIcon/svgs/Dependent/Close.jsx#L16).

See [docs for componentWithName](https://tds.telus.com/components/index.html#/Utilities?id=section-proptypes), under 'It can optionally check for displayName'.